docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -744,14 +744,12 @@ Possible values are:
 
 | Network   | Chain   | Type        | Description                                                         |
 |-----------|---------|-------------|---------------------------------------------------------------------|
+| `mainnet` | Eth 2.0 | Production  | Main network.                                                       |
```

Comment:
The `pyrmont` network has been added as well.   It's another Multi-client testnet using the v1.0 spec and very similar to the Mainnet configuration.

Reply:
Done